sam/diploidIndel_test.go

```diff
@@ -0,0 +1,163 @@
+package sam
+
+import "testing"
+
+var DiploidIndelCallFromPileTests = []struct {
+	P                 Pile
+	Delta             float64
+	Epsilon           float64
+	Kappa             float64
+	ExpectedInsertion DiploidInsertion
+	ExpectedDeletion  DiploidDeletion
+}{
+	{P: Pile{CountF: [13]int{30, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}, //high coverage, clear complex het insertion
+		InsCountF: map[string]int{"AAT": 7, "AT": 6},
+		InsCountR: map[string]int{"AAT": 6, "AT": 5}},
+		Delta:             0.01,
+		Epsilon:           0.01,
+		Kappa:             0.05,
+		ExpectedInsertion: DiploidInsertion{Type: IaIb, Ia: "AAT", Ib: "AT"},
```

Comment:
Note to self: running insertion prediction and deletion prediction on the same pile should get the same/consistent results